packages/server/src/cli.ts

```diff
@@ -139,6 +140,22 @@ if (isCli) {
 		}
 	}
 
+	let newestVersion = "";
+	if (process.env.VERSION) {
+		let currentVersion = process.env.VERSION;
+		const recentRelease = await getRecentRelease();
+
+		if (compareVersions(currentVersion, recentRelease)! <= 1) {
```

Comment:
No need to compare versions. If the latest version !== the current version then an update is available.